pyvera/__init__.py

```diff
@@ -637,7 +637,10 @@ def get_color_index(self, colors, refresh=False):
         ci = None
         sup = self.get_complex_value('SupportedColors')
         if sup is not None:
-            ci = [sup.split(',').index(c) for c in colors]
+            try:
```

Comment:
This works, however the result doesn't tell you which index matched with incoming supported color list.
Example:
`
sup = 'A,I,G'
colors = ['R','G','B']
ci = [sup.split(',').index(c) for c in colors if c in sup.split(',')]
print(ci)
[2]
`
You know something matched at 2, but which one it was is unknown.
My vote would be to just catch any errors if something is not found in the SupportedColors list.  I suppose a 'proper' fix would be to add a get_supported_colors method, so you could first find out of RGB is even supported before trying to get the index of the colors.  Or modifying it so it would insert a `-1` or `None` if an index is unfound.
But if your lights support other color sets then RGB I'm not sure how that would correspond to Home-Assistant.
Is 'W,A,I' what your SupportedColors come back as?